dbos/_dbos_config.py

```diff
@@ -424,6 +407,28 @@ def process_config(
     # Check the connectivity to the database and make sure it's properly configured
     # Note, never use db wizard if the DBOS is running in debug mode (i.e. DBOS_DEBUG_WORKFLOW_ID env var is set)
     debugWorkflowId = os.getenv("DBOS_DEBUG_WORKFLOW_ID")
+
+    # Pretty-print where we've loaded database connection information from, respecting the log level
```

Comment:
Can we move the printing logic to where we actually load the database connection? Otherwise, we need to maintain the same logic in two places.

Reply:
I see. The logic makes it possible to load DB Host from env, DB User from other places, and password from yet another place... This can cause bugs.